shared/src/common/enums/api/bs-api-path.enum.ts

```diff
@@ -1,6 +1,7 @@
 enum BSApiPath {
   ROOT = '/',
   SPACES = '/spaces',
+  SPACE = '/spaces/:id',
```

Comment:
```suggestion
  SPACES_$ID = '/spaces/:id',
```
`$` for dynamic parameters